src/mixins/eraser_brush.mixin.js

```diff
@@ -929,6 +929,8 @@
         }
 
         var path = this.createPath(pathData);
+        //  needed for `intersectsWithObject`
+        path.setCoords();
```

Comment:
i m worried this won't work if there is some zoom level going on
But this for sure is fixing something, so for now is fine.